### PR TITLE
Finish updating headers to use template & textviz audit

### PR DESCRIPTION
**Pull request by rebeccabilbro**
This PR is towards #456 and #509 and targeted at the files that we haven't already polished via the visualizer audit (#669). This includes:

- making code header updates to ensure that the ID line will be populated when we run `toto` (our codehead script) &mdash; which we'll do right before we issue v1.0
- applying `black` formatting to the files

@bbengfort one thing I realized as I was going through these is that we might also want a mini-audit of the `TextVisualizers`; so here it is:

- `text.base`
    - Defines `TextVisualizer`, which is a `Visualizer`
    - `fit` returns `self`
    - has one test, thin coverage 
- `text.dispersion`
    - Defines `DispersionPlot`, which is a `TextVisualizer`
    - `fit` calls `draw` and returns `self`, `draw` (now) returns `self.ax`
    - quick method is called `dispersion` and (now) calls `finalize` and (now) returns `visualizer`.
    - Overall looks ok, some docstrings are missing
    - has test coverage, uses hobbies corpus, has image comparison tests, has a quick method test
- `text.freqdist`
    - Defines `FreqDistVisualizer` (aliased to `FrequencyVisualizer`), which is a `TextVisualizer`
    - `fit` calls `draw` and returns `self`, `draw` (now) returns `self.ax`
    - quick method is called `freqdist` and (now) calls `finalize` and (now) returns `visualizer`.
    - Overall looks ok, some docstrings are missing
    - only one test (though it is an image comparison test), no quick method test, thin coverage overall
- `text.postag`
    - Defines `PosTagVisualizer`, which is a `TextVisualizer`
    - `fit` calls `draw` and returns `self`, `draw` returns `self.ax`
    - quick method is called `postag` and (now) calls `finalize` and (now) returns `visualizer`. 
    - has test coverage (Note: some tests require that `spacy` and `nltk` are installed), no YB datasets tests, has image comparison tests, has a quick method test
- `text.tsne`
    - Defines `TSNEVisualizer`, which is a `TextVisualizer`
    - `fit` calls `draw` and returns `self`, `draw` (now) returns `self.ax`
    - quick method is called `tsne` and (now) calls `finalize` and (now) returns `visualizer`. 
    - has test coverage, uses hobbies corpus, has pandas tests, has image comparison tests, no quick method test, several tests call sklearn's make_classification and generate a new dataset (no fixture) 
- `text.umap_vis`
    - Defines `UMAPVisualizer`, which is a `TextVisualizer`
    - `fit` calls `draw` and returns `self`, `draw` (now) returns `self.ax`
    - quick method is called `umap` and (now) calls `finalize` and (now) returns `visualizer`. 
    - has test coverage (Note: tests require that `umap` is installed), uses hobbies corpus, has pandas tests, has image comparison tests, no quick method test, several tests call sklearn's make_classification and generate a new dataset (no fixture)

### CODE REVIEW

File: yellowbrick/__init__.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # yellowbrick
 # A suite of visual analysis and diagnostic tools for machine learning.
 #
-# Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
+# Author:   Rebecca Bilbro
```
Comment (bbengfort) [inline yellowbrick/__init__.py]: @rebeccabilbro if possible, I'd like to add my name here as well

File: yellowbrick/bestfit.py [hunk 2]
```diff
@@ -95,11 +96,11 @@ def draw_best_fit(X, y, ax, estimator='linear', **kwargs):
 
     # Estimators are the types of best fit lines that can be drawn.
     estimators = {
-        LINEAR: fit_linear,               # Uses OLS to fit the regression
-        QUADRATIC: fit_quadratic,         # Uses OLS with Polynomial order 2
-        EXPONENTIAL: fit_exponential,     # Not implemented yet
-        LOG: fit_log,                     # Not implemented yet
-        SELECT_BEST: fit_select_best,     # Selects the best fit via MSE
```
Comment (bbengfort) [inline yellowbrick/bestfit.py]: Bleh, kind of annoying that Black did this - it usually lines up comments, but I guess not in a dictionary.

File: yellowbrick/datasaurus.py [hunk 3]
```diff
@@ -29,222 +29,1175 @@
...
-         17.04828941, 17.01565319, 17.06219214, 17.04110689, 17.13489391,
-         17.06772306, 17.16994971, 17.10571651, 16.75492389, 17.07814052,
-         17.08518438, 17.14760476, 16.90746981, 17.16234971, 17.24045586,
-         17.18019648, 17.10577072, 16.99296341, 17.08831585, 16.57271805,
-         17.22109553, 17.06474308, 17.0651685 , 17.07652235, 17.20885971,
-         17.20421434, 17.08465518, 17.09388377, 15.77189199, 17.00426226,
-         16.17493491, 17.03184749, 17.0049424 , 16.69484223, 17.04514941,
-         16.94292965, 16.94627981, 17.01958137, 50.16698595, 87.51396042,
-         83.99735692, 82.99075   ]])]
+    np.array(
```
Comment (bbengfort) [inline yellowbrick/datasaurus.py]: Ouch ... this is also not fun. I wonder if we should just put anscombe and datasaurus into `datasets`?

File: yellowbrick/text/dispersion.py [hunk 4]
```diff
@@ -289,26 +321,32 @@ def dispersion(words, corpus, y=None, ax=None, colors=None, colormap=None,
...
+        **kwargs
     )
 
     # Fit and transform the visualizer (calls draw)
     visualizer.fit(corpus, y, **kwargs)
+    visualizer.finalize()
 
-    # Return the axes object on the visualizer
-    return visualizer.ax
+    # Return the visualizer objec
```
Comment (bbengfort) [inline yellowbrick/text/dispersion.py]: Not worth changing and waiting for CI to run again - but just wanted to prove that I was doing a thorough review ;-)

File: yellowbrick/datasaurus.py (hunk 3 above)
Comment (rebeccabilbro) [reply yellowbrick/datasaurus.py]: Yeah sorry 🤕

File: yellowbrick/text/dispersion.py (hunk 4 above)
Comment (rebeccabilbro) [reply yellowbrick/text/dispersion.py]: good catch!

File: yellowbrick/__init__.py (hunk 1 above)
Comment (rebeccabilbro) [reply yellowbrick/__init__.py]: absolutely!

**Pull request merged**